src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+}
+
+body {
+  margin: 0 8px;
+}
+
+.field {
+  margin-top: 20px;
+  width: 100%;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  font-family: inherit;
+  color: inherit;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-sizing: border-box;
+  outline: none;
+}
+
+.field--big {
+  line-height: 68px;
+  font-size: 16px;
+  background-position: 26px;
+  padding-left: 62px;
+}
+
+.field--small {
+  line-height: 40px;
+  font-size: 14px;
+  background-position: 13px;
+  background-size: 11px;
+  padding-left: 33px;
+}
+
+.field:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.field:focus {
+  font-weight: bold;
+  background: url(images/Search.svg),
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-repeat: no-repeat, repeat;
+}
```

Comment:
It's better to add bold font styles for all inputs, and normal font styles for placeholders (use ::placeholder)